Update MANIFESTO.md

MairaDuqueA opened the pull request:
Adjunto pilares

MairaDuqueA committed:
Update MANIFESTO.md

leonfhl commented on _ES/MANIFESTO.md:
```diff
@@ -21,31 +21,7 @@ Hoy se acaba la educación que encasilla y empieza el aprendizaje que descubre d
-
-Estos pilares no son en un principio una creación de nuestro grupo si no más una recoleccion de diferentes movimientos como el Deeper Learning, Aprendizage Invertido, <poner mas>.
+•	**Cada persona crea sus conocimientos:** Sabemos que cada persona puede construir sus propios conocimientos a partir de las experiencias vividas, los seres con las que comparte, y las herramientas que tiene a su disposición. 
+•	**Habilidades para aprender y cambiar:** Queremos potenciar el desarrollo de habilidades que le permita a las personas pensar de manera crítica los retos que enfrentamos hoy como sociedad, resolver problemas creativamente y ser autodidactas.
```
Creo que en vez de _cambiar_ pondría algo como "Habilidades para aprender continuamente", pensando que en más que momentos de cambio las personas se preparan para continuamente aprender. Y en esta sección me gustaría inclui de alguna forma algo como "Habilidades para enseñar continuamente" de manera que el sistema sea autoescalable y más sostenible, además que creo que enseñar desde etapas tempranas y asi sea de forma corta y segmentada fortalece mucho el aprendizaje.

srcolinas replied:
- También me gusta más la frase "Habilidades para aprender continuamente", por lo que dice @leonfhl 
- Contradigo con @leonfhl  con el de enseñanza, creo que contradice el pilar de crear conocimiento.

leonfhl commented on _ES/MANIFESTO.md:
```diff
@@ -21,31 +21,7 @@ Hoy se acaba la educación que encasilla y empieza el aprendizaje que descubre d
-Estos pilares no son en un principio una creación de nuestro grupo si no más una recoleccion de diferentes movimientos como el Deeper Learning, Aprendizage Invertido, <poner mas>.
+•	**Cada persona crea sus conocimientos:** Sabemos que cada persona puede construir sus propios conocimientos a partir de las experiencias vividas, los seres con las que comparte, y las herramientas que tiene a su disposición. 
+•	**Habilidades para aprender y cambiar:** Queremos potenciar el desarrollo de habilidades que le permita a las personas pensar de manera crítica los retos que enfrentamos hoy como sociedad, resolver problemas creativamente y ser autodidactas.
+•	**Trabajo individual y colectivo:** Cada persona en su proceso de aprendizaje necesita su espacio solitario para descubrir sus propias rutas, preguntas y formas de aprender; pero compartir las preguntas y búsquedas con otros nos lleva a enriquecer la comprensión del mundo, de la vida, a encontrar rutas colectivas para sumar muchas manos a descubrir e implementar soluciones para transformación q ...
```
Creo que la parte final del "implementar soluciones para transformación que necesitamos." da un sentido de un problema actual y los pilares siento que deberían ser más atemporales. Lo cambiaría por algo tipo "...implementar soluciones que traigan bienestar a las comunidades", o a la humanidad.

an-jara replied:
Me pareció brutal todo! En el trabajo individual y colectivo, lo dejaría hasta "sumar muchas manos", pues ya el objetivo de la suma de estas manos es lo que justamente queremos, entonces creo que es un poco redundante.

srcolinas replied:
Yo lo dejaría así:

**Trabajo individual y colectivo:** Cada persona en su proceso de aprendizaje necesita su espacio solitario para descubrir sus propias rutas, preguntas y formas de aprender; pero compartir las preguntas y búsquedas con otros nos lleva a enriquecer la comprensión del mundo, de la vida.

an-jara commented on _ES/MANIFESTO.md:
```diff
@@ -21,31 +21,7 @@ Hoy se acaba la educación que encasilla y empieza el aprendizaje que descubre d
+•	**Cada persona crea sus conocimientos:** Sabemos que cada persona puede construir sus propios conocimientos a partir de las experiencias vividas, los seres con las que comparte, y las herramientas que tiene a su disposición. 
+•	**Habilidades para aprender y cambiar:** Queremos potenciar el desarrollo de habilidades que le permita a las personas pensar de manera crítica los retos que enfrentamos hoy como sociedad, resolver problemas creativamente y ser autodidactas.
+•	**Trabajo individual y colectivo:** Cada persona en su proceso de aprendizaje necesita su espacio solitario para descubrir sus propias rutas, preguntas y formas de aprender; pero compartir las preguntas y búsquedas con otros nos lleva a enriquecer la comprensión del mundo, de la vida, a encontrar rutas colectivas para sumar muchas manos a descubrir e implementar soluciones para transformación q ...
+•	**Conocimiento abierto:** Los espacios en los que suceden los procesos de aprendizaje son de acceso libre para cualquier persona desde cualquier lugar del mundo, y se alimentan permanentemente de la creación abierta a muchas manos.
```
Agregaría, después de "cualquier persona desde cualquier lugar del mundo": que quiera aprender o compartir aprendizaje con otros. Esto porque creo que el objetivo es que las personas puedan aprender aquí pero que también tengan herramientas para compartir aprendizaje para otros.

leonfhl committed:
Actualización a primera persona, parrafos

Revisar la parte (implementar soluciones para transformación que necesitamos.) debería ser otro pilar? Maira, Leon
Revisar un nuevo pilar de innovación y experimentación sobre el sistema de aprendizaje? Se encarga Andre

MairaDuqueA committed:
Update MANIFESTO.md
